tests/Functional/EventListener/InvalidationListenerTest.php

```diff
@@ -19,83 +19,6 @@ class InvalidationListenerTest extends WebTestCase
 {
     use MockeryPHPUnitIntegration;
 
-    public function testInvalidateRoute()
```

Comment:
do we have corresponding tests with attributes? if not we should convert this to test attributes so we have functional tests of the thing working in action.

Reply:
Yes. The test alreay exist on attribute.

Reply:
great!